test/katello_test_helper.rb

```diff
@@ -86,24 +86,23 @@ module FixtureTestCase
     Dir.mkdir(self.fixture_path)
     FileUtils.cp(Dir.glob("#{Katello::Engine.root}/test/fixtures/models/*"), self.fixture_path)
     FileUtils.cp(Dir.glob("#{Rails.root}/test/fixtures/*"), self.fixture_path)
+
+    taxonomies_file = "#{self.fixture_path}/taxonomies.yml"
+    taxonomies = YAML.safe_load(File.read(taxonomies_file))
+    taxonomies.values.each do |taxonomy|
+      next unless taxonomy['type'] == 'Organization'
+      taxonomy['label'] = taxonomy['name'].tr(' ', '_')
+    end
+    File.open(taxonomies_file, 'w') { |file| file.write(taxonomies.to_yaml) }
+
     fixtures(:all)
-    FIXTURES = load_fixtures(ActiveRecord::Base)
```

Comment:
Removing this wasn't necessary but it seems totally unnecessary to have it, and cleaned up some inconsistencies in using fixtures